### ISSUE TITLE
Issue running `ide-helper:generate` with Lumen v6.3.5 as a Composer  post-update-cmd

### ISSUE THREAD

**Issue by drew887** [barryvdh/laravel-ide-helper#1064]
### Versions:
- ide-helper Version: 2.8.1
- Lumen Version: v6.3.5
- PHP Version: 7.4

### Description:

Using above version of lumen with 2.8.1 of ide-helper causes the following issue when running composer update:
```
Writing lock file
Generating optimized autoload files
> @php artisan ide-helper:generate
[2020-09-18 19:24:50] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Application' not found in /vendor/barryvdh/laravel-ide-helper/src/IdeHelperServiceProvider.php:119

*** ALL OF THIS REMOVED FOR BREVITY ***


In IdeHelperServiceProvider.php line 119:

  Class 'Illuminate\Foundation\Application' not found
```

relevant section from composer.json
```
        "post-update-cmd": [
            "@php artisan ide-helper:generate",
            "@php artisan ide-helper:meta"
        ],
```

Just a rough guess https://github.com/barryvdh/laravel-ide-helper/pull/1026 is related in some way? its commit message 
```
TL;DR: can't use anything in the `Illuminate\Foundation` namespace
without safety check, as it doesn't exist in Lumen.
```
Seems to indicate this issue.

### Steps To Reproduce:
Simply running composer update with the above `post-update-cmd` config for composer

**Comment by mfn** [comment]
Duplicate of https://github.com/barryvdh/laravel-ide-helper/issues/1042

PR with fix was been merged https://github.com/barryvdh/laravel-ide-helper/pull/1043 but unfortunately we've to wait for a new release

ping @barryvdh

**Comment by drew887** [comment]
Ah sorry, I thought I did a quick search but I guess i missed checking for closed issues sorry, I'll go ahead and close this one as well. :sweat_smile:
